### ISSUE TITLE
Subset of rules possible?

### ISSUE THREAD

**Issue by nekapoor** [benignware/schedulable#30]
If I'm only looking to have the "singular" and "weekly" options appear in the rules, is that possible?

**Comment by yshmarov** [comment]
Go to schedule_input.rb.

replace:
`        b.input(:rule, collection: ['singular', 'daily', 'weekly', 'monthly'], label_method: lambda { |v| I18n.t("schedulable.rules.#{v}", default: v.capitalize) }, label: false, include_blank: false) << 
`
with
`        b.input(:rule, collection: ['singular', 'weekly'], label_method: lambda { |v| I18n.t("schedulable.rules.#{v}", default: v.capitalize) }, label: false, include_blank: false) << 
`

**Comment by rexblack** [comment]
Good point. This should be a configuration option, named `rules` for example. Hopefully i got some time soon to fix some issues. Create a pull request if you like...
